packages/ui/locales/en/views.json

```diff
@@ -280,7 +283,11 @@
     "activityHistory": "Activity history",
     "createNew": "Create new connector",
     "updated": "Last updated",
-    "lastConnected": "Last connected"
+    "lastConnected": "Last connected",
+    "viewConnectorDetails": "View connector details",
+    "connector": "Connector:",
+    "status": "Status:",
```

Comment:
Avoid `:` in translations, otherwise this cannot be used where we just need to have `Connector` or `Status`.

Reply:
fixed